### PR TITLE
Display images before blurbs on mobile.

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: app/views/static_pages/landing.html.haml [hunk 1]
```diff
@@ -42,20 +46,26 @@
...
               libraries such as Rails.
-          .col-md-6.col-md-pull-6
-            = image_tag('landing/landing-plan.png', alt: 'RubyBench Plan', class: 'landing-graphic-medium')
 
         .row.landing-section
-          .col-md-6
+          .col-md-6.col-md-push-6
+            = image_tag('landing/landing-hardware.png', alt: 'RubyBench Hardware', class: 'landing-graphic-medium')
+
+          .col-md-6.col-md-pull-6
```
Comment (tgxworld) [inline app/views/static_pages/landing.html.haml]: @yangshun Is this push pull considered hackish? xD

Comment (yangshun) [inline app/views/static_pages/landing.html.haml]: Nope, these classes are meant for reversing the order of rendering on mobile widths, exactly what you are trying to do here. On hindsight, I think the way you are doing now is better: images should be before the blurb. Oops.

File: app/views/static_pages/landing.html.haml [hunk 2]
```diff
@@ -42,20 +46,26 @@
               providing the percentage difference between commits and releases.
               In the near future, RubyBench will also monitor and track popular Ruby
               libraries such as Rails.
-          .col-md-6.col-md-pull-6
-            = image_tag('landing/landing-plan.png', alt: 'RubyBench Plan', class: 'landing-graphic-medium')
 
         .row.landing-section
-          .col-md-6
+          .col-md-6.col-md-push-6
+            = image_tag('landing/landing-hardware.png', alt: 'RubyBench Hardware', class: 'landing-graphic-medium')
```
Comment (yangshun) [inline app/views/static_pages/landing.html.haml]: Haha you can remove the brackets to be more consistent with your style XD

**Comment by yangshun** [comment]
LGTM :+1:

File: app/views/static_pages/landing.html.haml (hunk 1 above)
Comment (tgxworld) [inline app/views/static_pages/landing.html.haml]: Ah cool! Thanks!! :smile:

File: app/views/static_pages/landing.html.haml (hunk 2 above)
Comment (tgxworld) [inline app/views/static_pages/landing.html.haml]: heheh

**Pull request merged**